io.github.pieterdd.StretchBreak.yaml

```diff
@@ -0,0 +1,43 @@
+id: io.github.pieterdd.StretchBreak
+runtime: org.gnome.Platform
+runtime-version: '48'
+sdk: org.gnome.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+command: stretch-break
+finish-args:
+- "--share=ipc"
+- "--share=network"
+- "--socket=fallback-x11"
+- "--socket=wayland"
+- "--socket=pulseaudio"
+- "--device=dri"
+- "--env=G_MESSAGES_DEBUG=none"
+- "--env=RUST_BACKTRACE=1"
+- "--talk-name=org.gnome.Mutter.IdleMonitor"
+- "--talk-name=org.freedesktop.ScreenSaver"
+- "--talk-name=org.kde.ScreenSaver"
+- "--talk-name=org.freedesktop.Notifications"
```

Comment:
notifications don't go through the portal?

Reply:
I call `org.freedesktop.Notifications.Notify` to create and `org.freedesktop.Notifications.CloseNotification` to revoke previously created notifications. When I remove this `--talk-name` line, I don't see any notification.

Reply:
I had a problem with a library where it would not trigger notifications in certain conditions, so I started doing a `notify-send` call instead. Later I noticed a problem withdrawing the notification using CloseNotification when running from the Flatpak container, at which point I switched to doing `gdbus` calls for both Notify and CloseNotification.

I'll check if it works with libnotify.

Reply:
It does. I've removed the now-obsolete permission.